sdk/arm/compute-preview/ci.yml

```diff
@@ -0,0 +1,15 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
This CI pipeline is not taking effect, or I have to separate the preview module into another PR to start it?